docs/UserGuide.md

```diff
@@ -77,9 +76,86 @@ Profile | me | category, descriptors
 * Parameters can be in any order.<br>
   e.g. if the command specifies `i/INDUSTRY r/REQUIREMENT`, `r/REQUIREMENT i/INDUSTRY` is also acceptable.
 
-</div>
+## Features
+
+### Internship Application
+
+#### Adding an internship application: `add-int`
+
+Applies for an internship. This internship will be automatically added to the list of jobs in the company
+(if company has already been created). Otherwise, it creates the company and adds this job into it.
+
+Format: `add -int n/COMPANY_NAME j/JOB_NAME [i/INDUSTRY] [p/PERIOD] [w/WAGE] [r/REQUIREMENT]...`
+
+Examples:
+- add -int n/Google i/Software j/Software Engineer
+- add -int n/Google i/Software j/Software Engineer r/React w/3000 r/HTML5
```

Comment:
Should we markdown these lines with backticks as well, to put them in code format, as per the other examples?

Reply:
Good catch, updated it already. Thanks